config/routes.rb

```diff
@@ -1,7 +1,8 @@
 Rails.application.routes.draw do
   root to: 'pages#home'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  get '/skills', to: 'skills#index'
-  get '/skills/:id', to: 'skills#show'
-  get '/:tag_id/index', to: 'skills#index'
+  resources :skills, only: [ :home, :index, :show]
+
+  end
```

Comment:
There should be only one end in this file because you dont have a "do" so it isnt a block